### ISSUE TITLE
Settings for the best performance

### ISSUE THREAD

**Issue by SanPen** [ERGO-Code/HiGHS#1192]
Hi, 

I am worried that somehow I am not using Highs to all of its potential. For instance I'm running the optimization on a 32-thread computer and I get the following message:

`Using EKK parallel dual simplex solver - PAMI with concurrency of 8`

I have the threads setting to 0 (max, as I understand it) Is there a limitation to the number of threads?  

As a suggestion, a comprehensive list of settings and their interaction would be appreciated. I am aware of the GAMS [docs page](https://distdocs.gams.com/latest/docs/S_HIGHS.html), and it is not quite clear what the internal interaction between the settings are.

**Comment by jajhall** [comment]
The parallel performance of HiGHS is very limited at present. 
Parallel simplex is generally only a little faster, and will reduce beyond 4-8 threads. Indeed, since the parallel simplex solver lacks some features that I've added to improve the serial solver, it may even be slower. 
In the MIP solver, there is currently only a little parallelism - in the symmetry detection. A parallel tree search is in the pipeline, but development resources are limited.

So, in short, you're getting the best with the defaults.

As for documentation, it's been an embarrassment to us for a long time, but we've made big strides in the past two weeks. WIP is [here](https://ergo-code.github.io/HiGHS/)

**Comment by SanPen** [comment]
In my software, to deal with threading scalability I've been using [mimalloc](https://github.com/microsoft/mimalloc), getting about 30% better performance for free essentially.
